### ISSUE TITLE
Problem: calculateReward in /bounties runs a lot

### ISSUE THREAD

**Issue by janat08** [Blockrazor/blockrazor#791]
Problem: calculateReward is triggered every 10ms by session.get('now') in bounties.js and bountyRender.js, and it will trigger the whole autorun with lots of other operations that are in bounties.js.

Solution: In bounties.js split away calculateReward to operate on LocalBounties local collection with this.ids.get() filter, and adjust bounties helper query to sort by calculatedReward value of document.

**Comment by anbud** [comment]
I agree with this one, although it's low priority for now, it's not the happiest solution.

**Comment by gazhayes** [comment]
This is definitely one of the highest of the low priority tasks.

**Comment by gazhayes** [comment]
@janat08 did you want to do this one?

**Comment by janat08** [comment]
I'm going to remove sort field.

**Comment by gazhayes** [comment]
The UI can sort it by bounty amount anyway.

**Comment by janat08** [comment]
Tried doing it in 3 ways all failing: https://github.com/janat08/blockrazor/tree/calculateReward
I marked the relevant code as ``//try 3 at #791``
try 1 and 2 don't respect sort order for some reason with NaN values cropping up even if I overwrite them to 0. Try 1 is unreactive, try 2 isn't necessarily faster and tries 2 and 3 are bonkers although it's likely something else to blame.

@anbud what did you have in mind?

**Comment by anbud** [comment]
Oh, I didn't have any particular solution in mind, I just agreed that the current solution could be somehow optimized probably. 
I'll check your code to see if I see something wrong.

**Comment by janat08** [comment]
I think the LocalBounty.findOne part in autorun that consolidated all collection rerun causing a loop and deleting a value for calculateReward in try 1, but then why it also messed with try 2 I'm unclear.

**Comment by janat08** [comment]
The above worked.

**Comment by janat08** [comment]
It's noticeably more lively compared to master at x6 cpu throttle.

**Comment by janat08** [comment]
well yea, its  over %25 faster, but not the fastest option as it interfaces with minimongo which is what's used to diff for reward changes.
